tests/test_server_basic.py

```diff
@@ -1009,3 +1009,45 @@ def test_example_config(dispatcher_test_conf):
     print("\n\n\ntest_config_keys", test_config_keys)
 
     assert set(example_config_keys) == set(test_config_keys)
+
+
+def test_mosaic_image(dispatcher_live_fixture):
```

Comment:
it's a bit strange that "mosaic" is mentioned in generic dispatcher. It has concept of Image but not details like mosaic. But it's minor.

Reply:
I am not too familiar with these concepts and the relative terminology.

So just using image would be enough? Whreas mosaic is a more instrument-specific terminology?

Reply:
It depends. But what is relevant here is what is already present in dispatcher. You can see that there is `ImageQuery` and `ImageProduct` but no mosaics. It's a reasonable choice to introduce image but not mosaic.